src/worker.rs

```diff
@@ -104,6 +108,18 @@ fn list_files(logger: &Logger, name: &str, recursive: bool) -> Result<Vec<File>,
                                             err,
                                         )
                                     })?);
+
+                                    let get_sha = Command::new("svn")
```

Comment:
This will result in the command being called for every single revision, not every unique one.

Reply:
Should I use a map that stores the unique revisions instead?

Reply:
What would the values be? Use a set.

Reply:
So do I just not get the sha of a revision that already has a sha that was gotten?